doc/modules/http.websocket.md

```diff
@@ -53,6 +53,15 @@ The opcode `0x1` will be returned as `"text"` and `0x2` will be returned as `"bi
 
 Iterator over [`websocket:receive()`](#http.websocket:receive).
 
```

Comment:
Please prefix with a new level 4 heading and explicit anchor.
see an example at https://github.com/daurnimator/lua-http/blob/master/doc/modules/http.client.md#example-httpclientconnect-example